libdnf/rpm/package_query.cpp

```diff
@@ -722,30 +735,50 @@ void PackageQuery::filter_nevra(const libdnf::rpm::Nevra & pattern, libdnf::sack
 }
 
 void PackageQuery::filter_nevra(const PackageSet & package_set, libdnf::sack::QueryCmp cmp_type) {
-    if (cmp_type != sack::QueryCmp::EQ && cmp_type != sack::QueryCmp::NEQ) {
-        libdnf_throw_assert_unsupported_query_cmp_type(cmp_type);
+    libdnf_assert_same_base(p_impl->base, package_set.get_base());
+
+    bool cmp_not = (cmp_type & libdnf::sack::QueryCmp::NOT) == libdnf::sack::QueryCmp::NOT;
```

Comment:
Minor - to ensure that there is not GT + NOT (or similar) as comperator I would suggest to change the test:
```
bool cmp_not = (cmp_type == libdnf::sack::QueryCmp::NEQ);
if (cmp_not) {
    cmp_type = libdnf::sack::QueryCmp::EQ;
}
```

Reply:
Actually I plan to use LTE + NOT in list command implementation. Let me improve the description of the function.

Reply:
I've rephrased the doc string of `filter_nevra(PackageSet)` filter.